apps/src/craft/code-connection/levels.js

```diff
@@ -45,25 +45,23 @@ module.exports = {
           craftBlock('decoration') +
           craftBlock('getnameof') +
           craftBlock('getdataof') +
-          craftBlock('createblock', `<value name='BLOCKTYPE'>${block('text')}</value><value name='BLOCKDATA'>${block('text')}</value>`)) +
+          craftBlock('createblock')) +
       category('Blocks',
         craftBlock('tptotarget') +
         craftBlock('tptopos') +
         craftBlock('fill') +
         craftBlock('give') +
         craftBlock('kill') +
-        craftBlock('setblock', `<value name='ITEM'>${craftBlock('block')}</value>`) +
+        craftBlock('setblock') +
```

Comment:
That was a desire to create a consistent experience between the different visual scripting services that we support. There was some back-and-forth on it, but the task that was handed down to me essentially read: "We don't want some blocks to have default arguments and others to not have them, so lets make nothing have them".

I'm actually going to be arguing in -favor- of default arguments today in a meeting, but their removal was the current decision.